src/unix/notbsd/android/mod.rs

```diff
@@ -727,74 +712,6 @@ pub const NLA_TYPE_MASK: ::c_int = !(NLA_F_NESTED | NLA_F_NET_BYTEORDER);
 pub const SIGEV_THREAD_ID: ::c_int = 4;
 
 f! {
-    pub fn sigemptyset(set: *mut sigset_t) -> ::c_int {
```

Comment:
These are removed purposely. The old tests used android-9 api, which do not have this functions. The next api version distributed with android ndk is android-12 (release in 2011 https://en.wikipedia.org/wiki/Android_version_history), which define these functions. The definitions used by android are the same of others unix (src/unix/mod.rs).

I can see that this can be a breaking change... Should I return these implementations? (this will require moving definitions out of src/unix/mod.rs)

Keeping our own implementations of these functions seems odd.